test/python/qdmi/qiskit/test_qdmi_qiskit_gate_support.py

```
@@ -0,0 +1,225 @@
+    qc.measure([0, 1], [0, 1])
+
+    job = na_backend.run(qc, shots=100)
+    counts = job.get_counts()
+    assert sum(counts.values()) == 100
+
+
+def test_map_operation_returns_none_for_unknown() -> None:
+    """Test that unknown operations return None."""
+    backend = QiskitBackend()
+
+    assert backend._map_operation_to_gate("unknown_gate") is None  # noqa: SLF001
```

Ruff (0.14.2)

Unused `noqa` directive (non-enabled: `SLF001`)

Remove unused `noqa` directive

(RUF100)